build.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
+set -e
 
 if hash powershell 2>/dev/null; then
     echo 'Continuing with `powershell -noprofile -c Start-PSBuild`'
     powershell -noprofile -c "Import-Module ./build.psm1; Start-PSBuild"
 else
-    echo 'No `powershell`, see docs/building/linux.md or macos.md to build PowerShell!'
+   echo 'Continuing with full manual build'
+   ./build-from-scratch.sh
```

Comment:
Why? This makes no sense. The user has typed "build.sh". They expect the build to run. The script has proved to itself that powershell is not available. So now it should fail and tell the user what to do next? That seems to be the height of silly.